spec/features/cart/user_can_add_items_to_cart.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.feature "When a customer visits the items index and adds items to their cart", type: :feature do
+  before(:all) do
+    item1, item2 = create_list(:item, 2)
+  end
+
+  scenario "the cart counter says 1 after adding the first item" do
+    visit items_path
+    click_on ("Add Item", match: :first)
```

Comment:
Depending on how we set up our item divs, match: :first may give us an ambiguous match.  We may need to add a class to the Add Item buttons so that we can individually select them (e.g. add-item-1).